libpod/container.go

```diff
@@ -996,3 +996,13 @@ func (c *Container) IsInfra() bool {
 func (c *Container) IsReadOnly() bool {
 	return c.config.Spec.Root.Readonly
 }
+
+// NetworkDisabled returns whether the container is running with a disabled network
+func (c *Container) NetworkDisabled() bool {
+	for _, ns := range c.config.Spec.Linux.Namespaces {
```

Comment:
I fhink that we can return true if c.config.CreateNetNS is true at the beginning - that means we will create a network namespace even if one is not present

Reply:
We create a network namespace when the user specifies NONE, also, so wouldn't this be false.